### PR TITLE
Added readonly option to input and preview improvements.

### LINKED ISSUE #807: Input field does not have readonly option.
## Are you reporting a bug or a feature?

 - [ ] Bug
 - [x] Feature

## Expected Behavior

Input field should have readonly option, and be focusable when readonly.

## Actual Behavior

At the moment there is only disabled option which does not submit when the form is submitted and is not focusable.

### PR DESCRIPTION
**Pull request by nakovskijosif**
Fixes #807 

Changes proposed in this pull request:
Added readonly option to a-input.
 -
 -
 -

## Type of change

- [x] New feature (non-breaking change which adds functionality)
 
 # Checklist:
 
 - [ ] My code follows the style guidelines of this project
 - [ ] I have performed a self-review of my own code
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
 - [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/a-input/_preview.html [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
-<axa-input value="Pre-filled content" name="test-name" error></axa-input>
+<axa-input value="Pre-filled content - Disabled" name="test-name" disabled></axa-input>
+<axa-input value="Pre-filled content - Error" name="test-name" error></axa-input>
 <axa-input placeholder="Pre-filled content" name="test-name"></axa-input>
 <axa-input value="Pre-filled content" name="test-name" valid></axa-input>
 <axa-input value="Pre-filled content" name="test-name" valid inline></axa-input>
 <axa-input value="Pre-filled content" name="test-name" valid inline></axa-input>
 <axa-input value="Pre-filled content" name="test-name" valid inline></axa-input>
-<axa-input placeholder="Disabled paste" disable-paste name="test-name"></axa-input>
+<axa-input placeholder="Pre-filled content - Disabled paste" disable-paste name="test-name"></axa-input>
```
Comment (AndyOGo) [inline src/components/a-input/_preview.html]: @nakovskijosif 
@LucaMele 
I wonder what `disable-paste` is supposed to be, it is only defined as boolean proptype `disablePaste` below, but without any usage 🤔 

If you agree I would suggest to remove it.

Comment (nakovskijosif) [reply src/components/a-input/_preview.html]: This is a feature I created a while ago, disables paste on the input field. I only modified the preview option a bit on this PR.

File: src/components/a-input/_template.js [hunk 2]
```diff
@@ -20,7 +21,8 @@ export default function ({
           placeholder="${placeholder}"
           value="${value}"
           disabled="${disabled}"
-          autocomplete="${autocomplete}">
+          autocomplete="${autocomplete}"
+          readonly="${readonly}">
```
Comment (AndyOGo) [inline src/components/a-input/_template.js]: awesome

File: src/components/a-input/_preview.html (hunk 1 above)
Comment (AndyOGo) [reply src/components/a-input/_preview.html]: Ah I see, thanks for clarifying:)

I found the implementation here:
https://github.com/axa-ch/patterns-library/blob/cc8a91c3b4fd63d757de051a04651b0169fa8749/src/components/a-input/js/input.js#L14-L40

**Comment by nakovskijosif** [comment]
Thank you for the swift response.

**Pull request merged**